Thai/Songs/callher/callher.md

```diff
@@ -0,0 +1,19 @@
+คุณคิดว่าคุณอยากจะโทรหาเธอขึ้นมา แต่พอจะโทรจริง ๆ คุณกลับหาเบอร์โทรศัพท์
+ของเธอในเครื่องของคุณไม่เจอ สงสัยคุณคงจะลบมันไปนานแล้ว แต่เพราะคิดไว้
+ว่าจะโทรคุณเลยค้นดูในสมุดบันทึกเก่า ๆ สักพักก็พบ แล้วคุณก็โทรหาเธอ
+
+กริ๊ง...กริ๊ง...กริ๊ง...
+"สวัสดีค่ะ เพ็ญนภารับสายค่ะ" เสียงปลายสายตอบรับโทรศัพท์
+"สวัสดีครับเพ็ญ นี่ผมกฤษนะ เป็นไงบ้าง" คุณทักทาย
+"ว่าไงกฤษ เราสบายดี คิดยังไงถึงได้โทรมาล่ะเนี่ย ไม่ได้เจอกันตั้งหลายปี เป็นไงบ้าง" เธอถามกลับ
+"เราสบายดี พอดีเราฟังเพลงเก่าแล้วคิดถึงเลยโทรมาหา แฟนเธอคงไม่ว่ามั้ง" คุณออกตัว คุณรู้ดีว่าเธอแต่งงานไปหลายปีแล้ว ลูกกำลังน่ารัก
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.